### PR TITLE
Disaster Recovery documentation

### PR DESCRIPTION
**Pull request by emileswarts**

### CODE REVIEW

File: documentation/disaster-recovery.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Disaster Recovery
+
+Any unplanned downtime needs to be addresses as quickly as possible to minimise disruption of the end user devices.
```
Comment (neilkidd) [inline documentation/disaster-recovery.md:3]:
```suggestion
Unplanned downtime needs to be addressed as soon as possible to minimise disruption of end user devices.
```

File: documentation/disaster-recovery.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+Below is a list of potential causes, how these errors will be raised as alarms and how to remediate them:
```
Comment (neilkidd) [inline documentation/disaster-recovery.md:5]:
```suggestion
A number of potential scenarios, how these errors will be notified as alarms and remediation steps are detailed below:
```

**Review by neilkidd — COMMENTED** [review]
WIP

File: documentation/corsham-test.md [hunk 3]
```diff
@@ -28,38 +29,38 @@ This is integrated with the production MoJ network so will only work on our prod
 
 3. Change the permission of the pem file by running:
 
-```bash
-chmod 600 corsham_test.pem
-```
+    ```bash
+    chmod 600 corsham_test.pem
+    ```
```
Comment (jbevan4) [inline documentation/corsham-test.md:34]: Assume we want this linting style

File: documentation/disaster-recovery.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+A number of potential scenarios, their configured alarms and associated remediation steps are detailed below.
+
+1. [Corrupt configuration file](#corrupt-configuration-file)
+2. [Corrupt container was published](#corrupt-container-was-published)
+3. [Misconfigured infrastructure](bad-infrastructure-apply-with-terraform)
+4. [DHCP Subnet gets full](#dhcp-subnet-gets-full)
+5. [Services Overloaded](#services-overloaded)
+6. [AWS Availability Zone goes down](#aws-availability-zone-goes-down)
+7. [Other AWS failures](#other-aws-failures)
```
Comment (emileswarts) [inline documentation/disaster-recovery.md:13]: Need to delete this heading

File: documentation/disaster-recovery.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
+# Disaster Recovery
+
+Unplanned downtime needs to be addressed as soon as possible to minimise disruption of end user devices.
+
+A number of potential scenarios, their configured alarms and associated remediation steps are detailed below.
+
+1. [Corrupt configuration file](#corrupt-configuration-file)
+2. [Corrupt container was published](#corrupt-container-was-published)
+3. [Misconfigured infrastructure](bad-infrastructure-apply-with-terraform)
```
Comment (emileswarts) [inline documentation/disaster-recovery.md:9]: Dead link here

File: documentation/disaster-recovery.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+4. [DHCP Subnet gets full](#dhcp-subnet-gets-full)
```
Comment (emileswarts) [inline documentation/disaster-recovery.md:10]: Dead link

**Pull request merged**